code/obj/flock/furniture.dm

```diff
@@ -77,6 +79,8 @@
 /obj/stool/chair/comfy/flock/New()
 	..()
 	setMaterial(getMaterial("gnesis"))
+	APPLY_ATOM_PROPERTY(src, PROP_ATOM_FLOCK_THING, src)
+	src.AddComponent(/datum/component/flock_protection)
```

Comment:
```suggestion
	src.AddComponent(/datum/component/flock_protection, report_unarmed=FALSE)
```
If I'm not wrong, I think currently will make it so that swiveling a Flock chair marks you as an enemy?

Also, do think that report_attack = FALSE should be here. Was discussed before in Discord that deconstructing Flock tables/chairs is more of a repurposing thing rather than a damaging thing? Also, if not used, makes a small discrepancy where you can deconstruct Flock tables but not Flock chairs

Reply:
I'll add `report_attack=FALSE` also